src/xcode/ENA/ENA/Source/Views/GradientView.swift

```diff
@@ -198,6 +203,15 @@ class GradientView: UIView {
 			gradientLayer.startPoint = CGPoint(x: 0.0, y: 0.0)
 			gradientLayer.endPoint = CGPoint(x: 1.0, y: 1.0)
 			
+		case .solidGreen:
```

Comment:
It's a gradient, I would rather rename the other greens to solidLightGreen and solidDarkGreen

```suggestion
		case .green:
```

Reply:
👍 done